datastream-server/src/main/java/com/linkedin/datastream/server/EventProducer.java

```diff
@@ -200,13 +200,24 @@ private void validateEventRecord(DatastreamProducerRecord record) {
     }
   }
 
+  @Override
+  public void broadcast(DatastreamProducerRecord record, SendCallback sendBroadcastCallback) {
```

Comment:
Seems there is no way for caller to know whether some/all/none of the msgs succeeded/failed.